openedx/platform-plugin-aspects
# feat: adds endpoints for fetchGuestToken

Closes https://github.com/openedx/platform-plugin-aspects/issues/23

**Testing instructions**

1. Load the Instructor Dashboard
1. Ensure you can continue to use Superset uninterrupted for 5 min.
1. Add the SupersetXBlock to a course
1. Ensure you can continue to use Superset within the XBlock in the LMS and Studio for 5 min.

**Merge checklist:**
Check off if complete *or* not applicable:
